tests/acceptance/variant_view.js

```diff
@@ -221,8 +221,8 @@ test.describe('Variant View - ss by position', function() {
         config.shutdownDriver(driver);
     });
 
-    var expectedResults = [{"Organism": "Human", "Assembly": "GCA_000001405.14", "Submitter Handle": "", "Contig": "1",
-                            "Start": "3000017", "End": "3000017", "Reference": "C", "Alternate": "T", "ID":"ss1289423512",
+    var expectedResults = [{"Organism": "Human", "Assembly": "GCA_000001405.1 (GRCh37)", "Submitter Handle": "", "Contig": "",
+                            "Chromosome":"1", "Start": "3000017", "End": "3000017", "Reference": "C", "Alternate": "T", "ID":"ss1289423512",
```

Comment:
Also, as a general comment, tests are not working again. Could you please verify that at least they're passing locally on your machine?

Reply:
Nope, I didn't try to run any tests on my machine, so I don't have any suspicions on them failing, was asking just in case (specifically because of some Travis failures)